# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

BlockDetectionActionGoal action_goal
BlockDetectionActionResult action_result
BlockDetectionActionFeedback action_feedback
